src/validation/entities/schemas/signature.schema.ts

```diff
@@ -0,0 +1,10 @@
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
+
+function isSignature(value: `0x${string}`): boolean {
+  // We accept proposals of singular or concatenated signatures
+  return (value.length - 2) % 130 === 0;
```

Comment:
Nit: we have a `SIGNATURE_LENGTH` constant in other places, should we extract it to a common place and import it where it's used?

Reply:
As the mentioned `SIGNATURE_LENGTH` variables are located in tests and the growing size of the PR, I'd suggest we address it in a dedicated follow up where we centralise all usage of `r`, `s`, `v` and signature lengths to variables. What do you think?

Reply:
It sounds good to me, thanks!